src/promptflow/promptflow/_sdk/operations/_flow_operations.py

```diff
@@ -835,12 +834,18 @@ def _generate_flow_meta(
             # No flow meta for DAG flow
             return {}
 
+        # TODO: is it possible that there is no flow.path?
```

Comment:
I think flow.path = yaml file path for dag flow or script path for eager flow
Did we find any case no flow.path? We might add some asserts to replace the TODO comment